### ISSUE TITLE
sizeof/count is not implemented

### ISSUE THREAD

**Issue by dave-irvine** [asmblah/uniter#36]
```
var php = require('uniter').createEngine('PHP');
php.getStdout().on('data', function (text) { console.log(text); });
php.getStderr().on('data', function (text) { console.error(text); });

php.execute('<?php print sizeof([]);');
```

---

```
PHP Fatal error: Call to undefined function sizeof()
```

**Comment by asmblah** [comment]
Thanks for raising this @dave-irvine, I've now implemented `count(...)` in PHPRuntime `v4.4.0` (https://github.com/uniter/phpruntime/commit/66b4cca74ed074dc269a697f3e492a86c4ac947d)

Cheers!

**Comment by bkidwell** [comment]
This commit is in lots of recent tags and in `master` branch of `uniter/phpruntime`. Should the issue on this page be closed?

**Comment by asmblah** [comment]
Hi @bkidwell, thanks for the question! `count(...)` is indeed implemented as you say, but I've just checked and its alias `sizeof(...)` is not, so as that's the one mentioned in the title and description above I'll leave this open just until that's resolved and then I think we can close it.

**Comment by asmblah** [comment]
Implemented in uniter/phpruntime#10.
